### DISCUSSION TITLE
Is it possible to add comments?

### DISCUSSION THREAD

**Discussion by zachwolfe** [vvvv/SVG#1017] [Q&A]
Is it possible to add an XML comment `<!-- like this -->` as a child of an existing `SvgElement`? My current workaround is to just create an `SvgUnknownElement` with a "comment" element name, but that isn't ideal.

**Comment by mrbean-bremen** [comment]
As far as I know, there is no specific support for this, and your solution is probably the best one at the moment.

**Reply by zachwolfe** [reply]
Okay, that’s what I suspected, but thanks for getting back to me.
